src/context/AppProvider/AppContext.tsx

```diff
@@ -304,7 +305,7 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
       },
       // once the portfolio is loaded, fetch market data for all portfolio assets
       // and start refetch timer to keep market data up to date
-      enabled: portfolioLoadingStatus !== 'loading',
+      enabled: !isConnected || portfolioLoadingStatus !== 'loading',
```

Comment:
*do* fetch market data despite portfolio not being loaded, required for assets page